ui/qt-lib/src/Ariadne/UI/Qt.hs

```diff
@@ -76,6 +80,17 @@ runUIEventLoop eventIORef dispatcherIORef uiWalletFace historyFace putPass langF
 
     QCoreApplication.exec
 
+killQtUI :: SomeException -> IO ()
+killQtUI e = do
+  msg <- QMessageBox.new
+  QMessageBox.setWindowTitle msg ("Error" :: String)
+  QMessageBox.setIcon msg QMessageBox.Critical
+  QMessageBox.setText msg ("Got exception from backend: " <> show e :: String)
+  connect_ msg QMessageBox.buttonClickedSignal $ \_ -> QDialog.accept msg
+  connect_ msg QDialog.acceptedSignal $ QCoreApplication.exit 1
+  connect_ msg QDialog.rejectedSignal $ QCoreApplication.exit 1
+  void $ QDialog.exec msg
```

Comment:
Do you realy need this signal stuff? AFAIU, `QMessageBox.critical` would block until user closes it and after that you can just do `QCoreApplication.exit`.

Reply:
Fair enough

Reply:
Well, do you really need `QMessageBox.new` now? :)

Reply:
Your suggestions is to make `void $ QMessageBox.critical widget ("Error" :: String) ("Got exception from backend: " <> show e :: String)`? It requires parent widget anyway